src/gui/rss/feedlistwidget.cpp

```diff
@@ -40,6 +40,28 @@
 #include "base/rss/rss_session.h"
 #include "gui/uithememanager.h"
 
+namespace
+{
+    QIcon loadIcon(const QString &path, const QString &fallbackId)
+    {
+        const QPixmap pixmap {path};
+        if (!pixmap.isNull())
+            return QIcon(pixmap);
+
+        return UIThemeManager::instance()->getIcon(fallbackId);
+    }
+
+    QIcon rssFeedIcon(RSS::Feed *feed)
```

Comment:
`const RSS::Feed *feed`
or even `const RSS::Feed &feed` to avoid potential `nullptr`.